### PR TITLE
Broke error handling up into three elements for experiment when check…

### PR DESCRIPTION
**Pull request by jayich**
…ing for scriptscanner, parametervault, and context.

Review: @aransfor , @gregllong 

I made the error handling here explicit.

If these look good, I'll merge them in.

### CODE REVIEW

**Comment by gregllong** [comment]
Nice! LGTM

File: lib/servers/abstractservers/script_scanner/experiment.py [hunk 1]
```diff
@@ -21,17 +21,20 @@ def _connect(self):
             try:
                 self.cxn = labrad.connect()
             except Exception as e:
-                raise Exception ("Not able to connecto LabRAD")
+                raise Exception("Not able to connect to LabRAD")
+        try:
+            self.sc = self.cxn.servers['ScriptScanner']
+        except Exception as e:
```
Comment (gregllong) [inline lib/servers/abstractservers/script_scanner/experiment.py]: might be good to specify error type. For this line I think KeyError should be appropriate.

Comment (gregllong) [inline lib/servers/abstractservers/script_scanner/experiment.py]: For more details, see:

https://docs.python.org/2/tutorial/errors.html#handling-exceptions

**Comment by jayich** [comment]
@aransfor ,

Please review the most recent commit.

**Comment by aransfor** [comment]
LGTM

**Pull request merged**